pallets/nft/src/lib.rs

```diff
@@ -58,6 +60,8 @@ pub type CID = Vec<u8>;
 
 pub type HashByte32 = [u8; 32];
 
+pub const CREATION_FEE: u32 = 100;
```

Comment:
if it just used for testing, we can put it in mock file.

Reply:
Currently it's not possible because feature runtime benchmarks also uses this

Reply:
There is perhaps a better way, but I am not sure how to

Reply:
it is fine if benchmark also need it.